package.json

```diff
@@ -11,10 +11,13 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@types/react-modal": "^3.13.1",
```

Comment:
@Adegbite1999 in case you installed to types manually, we should probably remove them from `dependencies` and instead install them on `devDependencies`